src/GRA.Data/Model/Site.cs

```diff
@@ -47,6 +47,7 @@ public class Site : Abstract.BaseDbEntity
         public bool SinglePageSignUp { get; set; }
 
         public int? MaxPointsPerChallengeTask { get; set; }
+        public int? MaxPointsPerTrigger { get; set; }
```

Comment:
While challenges were done as a property on Site in probably makes more sense to this to be in the SiteSetting table. MaxPointsPerChallengeTask could/should be moved over as well since this PR makes changes to the Challenges max points already.

Reply:
Resolved in commits 4b5b38738a9a9570883379938660ae3f0c00e5ab, 545f2d6ce99faf187fca5f2a966b6a9c698acad8, and 6e387f8db24e6a4cd5523cd7271c74ec9d3d61b7